src/sql/strong-oracle/disable.sql

```diff
@@ -0,0 +1,119 @@
+
```

Comment:
I think so, but let's do that in a future refactoring so we can merge this before my vacation